File: src/main/java/dev/hiitsdevin/pastelwonderland/FabricTags.java
package dev.hiitsdevin.pastelwonderland;

import net.fabricmc.fabric.api.tag.TagRegistry;
import net.minecraft.block.Block;
import net.minecraft.tag.Tag;
import net.minecraft.util.Identifier;

public class FabricTags {
    public static final Tag<Block> DIRT = blockTagRegister("dirt");
    public static Tag<Block> blockTagRegister(String id) {
        return TagRegistry.block(new Identifier("c", id));
    }
}

File: src/main/kotlin/devos/pastelwonderland/Base/Organization/FabricTags.kt
package devos.pastelwonderland.Base.Organization

import net.fabricmc.fabric.api.tag.TagRegistry
import net.minecraft.block.Block
import net.minecraft.tag.Tag
import net.minecraft.util.Identifier

object FabricTags {
    val DIRT = blockTagRegister("dirt")
    fun blockTagRegister(id: String?): Tag<Block> {
        return TagRegistry.block(Identifier("c", id))
            // Bruh?
    }
}